package-lock.json

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
We're trying to not commit the package-lock.json

If you would be willing to just delete the file then commit it

Reply:
Done